client/components/tinymce/plugins/advanced/plugin.jsx

```diff
@@ -35,18 +35,18 @@ function advanced( editor ) {
 
 			toolbar.visible( isToolbarVisible );
 
-			if ( isAdvancedVisible ) {
-				if ( isSmallViewport ) {
-					containerPadding = Math.max( containerPadding, toolbar.getEl().clientHeight );
-				} else {
-					containerPadding += toolbar.getEl().clientHeight;
-				}
+			if ( isSmallViewport ) {
+				containerPadding = Math.max( containerPadding, toolbar.getEl().clientHeight );
+			} else {
+				containerPadding += toolbar.getEl().clientHeight;
 			}
 		} );
 
-		tinymce.DOM.setStyles( editor.getContainer(), {
-			'padding-top': containerPadding
-		} );
+		if ( editor.theme.panel.getEl().clientHeight ) {
+			tinymce.DOM.setStyles(editor.getContainer(), {
+				'padding-top': containerPadding
+			});
```

Comment:
>Always include extra spaces around elements and arguments:

https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#arrays-and-function-calls